backend/package.json

```diff
@@ -61,7 +61,7 @@
     "graphql-custom-directives": "~0.2.14",
     "graphql-iso-date": "~3.6.1",
     "graphql-middleware": "~3.0.2",
-    "graphql-shield": "~5.7.1",
+    "graphql-shield": "~5.6.1",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-24T17:49:34Z" title="Monday, June 24th 2019, 7:49:34 pm +02:00">Jun 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -61,7 +61,7 @@
     "graphql-custom-directives": "~0.2.14",
     "graphql-iso-date": "~3.6.1",
     "graphql-middleware": "~3.0.2",
-    "graphql-shield": "~5.7.1",
+    "graphql-shield": "~5.6.1",
```


downgraded because `5.7.1` was breaking the `CreatePost` implementation

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-29T17:09:06Z" title="Saturday, June 29th 2019, 7:09:06 pm +02:00">Jun 29, 2019</time>_

---


for all the lovely `PR` reviewers out there, there is this issue https://github.com/maticzav/graphql-shield/issues/408, which talks about what could be happening, and possible solutions